components/02-elements/rating/rating--rate.hbs

```diff
@@ -1,122 +1,51 @@
-                {{{ star.attributes }}}
-            >
-                <span class="rating__indicator {{ indicator.class }}" {{{ indicator.attributes }}}></span>
-            </label>
-            <input
-                type="radio"
-                name="ratings[4]"
-                id="Rating_2"
-                value="16"
-                class="radio__field"
-            >
-        </div>
-        <div
-            class="rating__rate-item {{ rateItem.class }}"
-            role="option"
-            aria-selected="false"
-            {{{ rateItem.attributes }}}
-        >
-            <label
-                for="Rating_3"
-                class="rating__star rating__star--rate {{ star.class }}"
-                aria-label="Rate option, 3 of 5. Click to vote"
-                {{{ star.attributes }}}
-            >
-                <span class="rating__indicator {{ indicator.class }}" {{{ indicator.attributes }}}></span>
-            </label>
-            <input
-                type="radio"
-                name="ratings[4]"
-                id="Rating_3"
-                value="16"
-                class="radio__field"
-            >
-        </div>
-        <div
-            class="rating__rate-item {{ rateItem.class }}"
-            role="option"
-            aria-selected="false"
-            {{{ rateItem.attributes }}}
-        >
-            <label
-                for="Rating_4"
-                class="rating__star rating__star--rate {{ star.class }}"
-                aria-label="Rate option, 4 of 5. Click to vote"
-                {{{ star.attributes }}}
-            >
-                <span class="rating__indicator {{ indicator.class }}" {{{ indicator.attributes }}}></span>
-            </label>
-            <input
-                type="radio"
-                name="ratings[4]"
-                id="Rating_4"
-                value="16"
-                class="radio__field"
-            >
-        </div>
-        <div
-            class="rating__rate-item {{ rateItem.class }}"
-            role="option"
-            aria-selected="false"
-            {{{ rateItem.attributes }}}
-        >
-            <label
-                for="Rating_5"
-                class="rating__star rating__star--rate {{ star.class }}"
-                aria-label="Rate option, 4 of 5. Click to vote"
-                {{{ star.attributes }}}
-            >
-                <span class="rating__indicator {{ indicator.class }}" {{{ indicator.attributes }}}></span>
-            </label>
-            <input
-                type="radio"
-                name="ratings[4]"
-                id="Rating_5"
-                value="16"
-                class="radio__field"
-            >
-        </div>
+        {{#each rateItems as |item|}}
+            <div
+                class="rating__rate-item {{ item.class }}"
+                role="option"
+                aria-selected="false"
+                {{{ item.attributes }}}
```

Comment:
IMO there is no need to use curly bracket. Check here and below ones

Reply:
I think we need triple curly bracket for attributes, I've tested it
![screen shot 2018-10-03 at 11 59 19](https://user-images.githubusercontent.com/15080214/46403829-da886000-c703-11e8-99bd-d169c91cb3d0.png)

Reply:
ok, sorry then